server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/model/Credentials.java

```diff
@@ -0,0 +1,7 @@
+package org.kaaproject.kaa.server.common.dao.model;
+
+import org.kaaproject.kaa.common.dto.HasVersion;
+import org.kaaproject.kaa.common.dto.credentials.CredentialsDto;
+
+public interface Credentials extends ToDto<CredentialsDto>, HasVersion {
```

Comment:
A credentials DTO represents some credentials – an instance of a class that implements this interface. Thus, the interface should provide getters for the fields contained in the DTO, so one can create the DTO from the credentials instance.